.gitignore

```diff
@@ -25,7 +25,6 @@ validators
 **/dist
 **/.nyc_output
 .tmp
-.vscode
```

Comment:
need to add
```
.vscode/launch.json
.vscode/settings.json
```

otherwise don't files would be tracked via git which I understand we don't want with this approach

and maybe also tasks.json?

Reply:
Good point